src/index.js

```diff
@@ -29,27 +20,19 @@
         }
     }
 
-    require('./modules/**/index.js', {mode: (base, files) => {
-        return files.map(module => {
-            return `
-                try {
-                    require('${module}');
-                } catch (e) {
-                    debug.error('Failed to ${module}', e.stack);
-                }
-            `;
-        }).join(' ');
-    }});
+    import('./modules/**/index.js');
```

Comment:
nit: let's make this + the regex check for `await import` instead so it's closer to the actual spec

Reply:
changes to await import